src/components/Registration/DefineRegistrationSummary.vue

```diff
@@ -85,7 +85,7 @@
       </template>
 
       <!-- Business Number -->
-      <template v-if="isTypePartnership">
+      <template>
```

Comment:
Just delete this pseudo-element then. Vue uses it for conditional logic only -- it doesn't render.